internal/flink/command_artifact_describe.go

```diff
@@ -48,6 +49,9 @@ func (c *command) describe(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	if _, err = c.V2Client.GetOrgEnvironment(environment); err != nil {
+		return fmt.Errorf("environment '%s' not found", environment)
```

Comment:
nit: I am not certain that 'not found' is the best error message for 5xx HTTP status codes, etc., but I am not very familiar with the CLI codebase, so it might be a common error message. 😁